test/k6/tests/find-tests.js

```diff
@@ -1,38 +1,92 @@
-import http from 'k6/http';
 import { browser } from 'k6/browser';
-import { sleep } from 'k6';
-
+import { check, sleep } from 'k6';
 export async function searchResultsTest(data) {
     const page = await browser.newPage();
 
     try {
         await page.goto(`${data.SETTINGS.baseUrl}`);
+        const startButton = await page.locator('//*[@id="main-content"]/div/div/a');
+
+        await Promise.all([startButton.click()]);
+        await page.locator('[name="postcode"]').type('E1 2EN');
+
+        const submitButton = await page.locator('//*[@id="main-content"]/div/div/form/button');
 
-        const startButton = await page.locator('input[class="govuk-button govuk-button--start"]').click();
+        await Promise.all([submitButton.click()]);
 
-        await Promise.all([page.waitForNavigation(), startButton.click()]);
+        const title = await page.locator('title').textContent();
+        await check(title, {
+            title: title === 'Services, groups and activities in this area (page 1 of 45) - Find support for your family - GOV.UK',
```

Comment:
question: is the 1 of 45 guaranteed? I can't remember if this is seeded data or not

Reply:
This isn't seeded data yet, before I run the test I'll need to seed service data for find then adjust this number. Good catch I'll put a todo